### DISCUSSION TITLE
Estimate monoenergetic image from single polyenergetic image

### DISCUSSION THREAD

**Discussion by Dale-Black** [JuliaImageRecon/Sinograms.jl#71] [Q&A] [answered]
When creating a scan of a digital phantom, it's relatively straightforward to create a polyenergetic image via integration. Is it possible to estimate a monoenergetic image once you have the polyenergetic image, while remaining in the image domain?

**Comment by JeffFessler** [comment]
The answer is likely "no" in general, but the question needs clarification first.
Linear attenuation is a function f(x; E) that is an energy-indexed collection of images.
So your meaning of "monoenergetic  image" presumably is just f(x; E_0) for some specific energy E_0.
But what is a "polyenergetic image"?

**Reply by Dale-Black** [reply]
By polyenergetic image, I mean a reconstructed image created from a polyenergetic xray spectrum

**Reply by JeffFessler** [reply, marked as answer]
Then probably you will need dual-energy or spectral CT for that unless your object happens to be just a single material of various densities.  Look at works like https://doi.org/10.1002/mp.12096

**Reply by Dale-Black** [reply]
Got it, yes this makes sense. I have read that paper as well as a few of yours. Just wanted to check before building out some simple simulations. Thank you!

**Reply by JeffFessler** [reply]
Just this morning I saw a presentation by GE that mentioned their "True Enhance" method that uses AI (of course) for this problem.  So maybe the answer is yes after all...

https://www.itnonline.com/content/ge-healthcare-highlights-commitment-delivering-better-patient-experiences-and-outcomes

**Reply by Dale-Black** [reply]
That is very interesting! I actually have a few follow up questions but they might be better suited over email. We (Imaging Physics Lab @ UCI) are beginning to dive into multi-material decomposition, which is one of your areas of expertise. Would you be willing to engage in an email thread with me and my advisor (Dr. Sabee Molloi, UCI), and if so, please do email me djblack@uci.edu

**Reply by Dale-Black** [reply]
Or if you'd rather, I would be happy to find your email online and email you directly first
